### PR TITLE
wip - DO NOT MERGE - Pipeline run test

### PR DESCRIPTION
**Pull request by CarolynMabbott**
# Changes

DO NOT MERGE

References: #161

Testing changes to ensure that the information received from the cluster is the same as whats expected

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -34,5 +36,196 @@ failed=0
...
+}
+
+
+
+#Fork port forward, once starts running never stops running until killed
+function port_forward() {
+    kubectl port-forward $(kubectl get pod -l app=tekton-dashboard -o name) 9097:9097
+}
+
+echo "Installing knative version $KNATIVE_VERSION"
```
Comment (vdemeester) [inline test/e2e-tests.sh]: Any reason for installing knative ?

Comment (CarolynMabbott) [reply test/e2e-tests.sh]: Was trying to `kubectl get ksvc` for something I was trying out and it required knative to work. If it turns out not to work then i'll get rid of the knative install

**Comment by CarolynMabbott** [comment]
Closing to do new pull request with only 1 commit

**Pull request closed without merging**